aiida_mlip/workflows/ht_workgraph.py

```diff
@@ -0,0 +1,126 @@
+"""Workgraph to run high-throughput calculations."""
+
+from pathlib import Path
+from typing import Callable, Union
+
+from aiida.engine import CalcJob, WorkChain
+from aiida_workgraph import WorkGraph, task
+from ase.io import read
+
+from aiida_mlip.helpers.help_load import load_structure
+
+
+@task.graph_builder(outputs=[{"name": "final_structures", "from": "context.structs"}])
+def ht_calc_builder(
+    calc: Union[CalcJob, Callable, WorkChain, WorkGraph],
```

Comment:
We haven't switched Aiida-mlip over to `__future__.annotations`, yet? Should at some point to align with Janus

Reply:
Good point, no, I probably should have added that when I added ruff. There's an issue now